tests/unit/QuantFile.spec.js

```diff
@@ -289,4 +288,49 @@ describe('QuantFile.vue', () => {
     })
   })
 
+  describe('tapestation X10 VDJ file', () => {
+    beforeEach(async () => {
+      quantFile = new cmp({propsData: { quant: 'library10XVDJ', filename: 'DN987654 - 2019-11-05 - 16-32-56-D5000_compactRegionTable(1).csv'}})
+      plate = fs.readFileSync('./tests/data/DN987654 - 2019-11-05 - 16-32-56-D5000_compactRegionTable(1).csv', 'ascii')
+      file = new File([plate], 'DN987654 - 2019-11-05 - 16-32-56-D5000_compactRegionTable(1).csv', { type: 'text/plain'})
+    })
+
+    it('will have a filename', () => {
+      expect(quantFile.filename).toEqual('DN987654 - 2019-11-05 - 16-32-56-D5000_compactRegionTable(1).csv')
+    })
+
+    it('will have a parsed filename', () => {
+      expect(quantFile.parsedFilename).toEqual('DN987654')
+    })
+
+    it('resolves', async () => {
+      expect.assertions(1)
+      await expect(quantFile.upload(file)).resolves.toEqual('File successfully uploaded')
+    })
+
+    describe('successful', () => {
+      beforeEach(async () => {
+        quantFile.upload(file)
+        await flushPromises()
```

Comment:
Wonder what the 2x flush is for? Is there a promise earlier not being resolved correctly

Reply:
I had the same thought.  I decided to just accept someone else found this works.